src/main/java/cs2103/v15_1j/jimjim/ui/JJUI.java

```diff
@@ -70,7 +70,8 @@ public DataLists getSearchResults(){
 		return tempList;
 	}
 
-	public void executeCommand(String userCommand){
+	public void executeCommand(String userCommand, DataLists displayList){
+		con.setDisplayList(displayList);
```

Comment:
again, as long as you don't change change `displayList` to refer to some other lists and only modify its content, you only need to do this once.